src/ast/astnodes.ts

```diff
@@ -118,167 +108,93 @@ const _attrs = ["lineno", "col_offset", "end_lineno", "end_col_offset"];
+export const Load = expr_context.Load;
+export const Store = expr_context.Store;
+export const Del = expr_context.Del;
 
 /* ----- boolop ----- */
-export class boolop extends AST {
-    static _name = "boolop";
+export enum boolop {
+    And = 3,
+    Or,
 }
-boolop.prototype._enum = true;
 
-export class AndType extends boolop {
-    static _name = "And";
-}
-export class OrType extends boolop {
-    static _name = "Or";
-}
-export const And = new AndType();
-export const Or = new OrType();
+export const And = boolop.And;
+export const Or = boolop.Or;
 
 /* ----- operator ----- */
-export class operator extends AST {
-    static _name = "operator";
-}
-operator.prototype._enum = true;
-
-export class AddType extends operator {
-    static _name = "Add";
-}
-export class SubType extends operator {
-    static _name = "Sub";
-}
-export class MultType extends operator {
-    static _name = "Mult";
-}
-export class MatMultType extends operator {
-    static _name = "MatMult";
-}
-export class DivType extends operator {
-    static _name = "Div";
-}
-export class ModType extends operator {
-    static _name = "Mod";
-}
-export class PowType extends operator {
-    static _name = "Pow";
-}
-export class LShiftType extends operator {
-    static _name = "LShift";
-}
-export class RShiftType extends operator {
-    static _name = "RShift";
-}
-export class BitOrType extends operator {
-    static _name = "BitOr";
-}
-export class BitXorType extends operator {
-    static _name = "BitXor";
-}
-export class BitAndType extends operator {
-    static _name = "BitAnd";
-}
-export class FloorDivType extends operator {
-    static _name = "FloorDiv";
-}
-export const Add = new AddType();
-export const Sub = new SubType();
-export const Mult = new MultType();
-export const MatMult = new MatMultType();
-export const Div = new DivType();
-export const Mod = new ModType();
-export const Pow = new PowType();
-export const LShift = new LShiftType();
-export const RShift = new RShiftType();
-export const BitOr = new BitOrType();
-export const BitXor = new BitXorType();
-export const BitAnd = new BitAndType();
-export const FloorDiv = new FloorDivType();
+export enum operator {
+    Add = 5,
+    Sub,
+    Mult,
+    MatMult,
+    Div,
+    Mod,
+    Pow,
+    LShift,
+    RShift,
+    BitOr,
+    BitXor,
+    BitAnd,
+    FloorDiv,
+}
+
+export const Add = operator.Add;
+export const Sub = operator.Sub;
+export const Mult = operator.Mult;
+export const MatMult = operator.MatMult;
+export const Div = operator.Div;
+export const Mod = operator.Mod;
+export const Pow = operator.Pow;
+export const LShift = operator.LShift;
+export const RShift = operator.RShift;
+export const BitOr = operator.BitOr;
+export const BitXor = operator.BitXor;
+export const BitAnd = operator.BitAnd;
+export const FloorDiv = operator.FloorDiv;
 
 /* ----- unaryop ----- */
-export class unaryop extends AST {
-    static _name = "unaryop";
+export enum unaryop {
+    Invert = 18,
```

Comment:
Why does this get a number?

Reply:
It was easier to do reverse lookup in the ast dump if the enum values were all different numbers.

Rather than Load, and Invert both being 0. Saying 18 here just says start from 18)

Or is the question why not use strings instead?

Reply:
Oh! ok that makes sense, I read this as just `Invert` being 18